rfhub2/cli/api_client.py

```diff
@@ -0,0 +1,79 @@
+from requests import session, post, get, delete, exceptions
+from typing import Dict, List
+
+from rfhub2.model import Collection, Keyword
+
+
+API_V1 = 'api/v1'
+TEST_COLLECTION = {"name": "healtcheck_collection", "type": "a", "version": "a", "scope": "a",
+                   "named_args": "a", "path": "a", "doc": "a", "doc_format": "a"}
+
+
+class Client(object):
+    """
+    API client with methods to populate rfhub2 application.
+    """
+    def __init__(self, app_url: str, user: str, password: str):
+        self.app_url = app_url
+        self.session = session()
+        self.api_url = f'{self.app_url}/{API_V1}'
+        self.session.auth = (user, password)
+        self.session.headers = {"Content-Type": "application/json", "accept": "application/json"}
+
+    def check_communication_with_app(self) -> None:
+        try:
+            self.get_collections()
+        except exceptions.RequestException:
+            print(f'Connection to application at {self.app_url} refused!\n'
+                  f'Check parameter for app_url.')
+            exit(1)
+        req_check_auth = self._post_request('collections', TEST_COLLECTION)
+        if req_check_auth['name'] == 'healtcheck_collection':
+            self._delete_request('collections', req_check_auth["id"])
+        else:
+            print('Check used credentials!')
+            exit(1)
+
+    def get_collections(self) -> List[Collection]:
```

Comment:
The type annotations here are misleading and AFAIR are displayed as warnings in PyCharm. Until we move to actually use here pydantic models, they should be changed to `Dict`

Reply:
It didn't highlight it, but of course I will switch to Dict